### PR TITLE
Update examples to Spine 0.10.26

### PR DESCRIPTION
**Pull request by mdrachuk**
This PR updates the existing examples to the latest Spine version. 

Summary: 
- Guava, gRPC and Protobuf updated to match the versions in Spine
- the package structure was migrated to `io.spine...`
- the examples using deprecated API were removed
- Google App Engine datastore example was fixed
- a fix for client subscription was added canceling them prior to shutdown

### CODE REVIEW

**Comment by mdrachuk** [comment]
@armiol, PTAL.

**Comment by mdrachuk** [comment]
@armiol, PTAL.

File: ext.gradle [hunk 1]
```diff
@@ -18,15 +18,23 @@
...
+    
+    guavaVersion = '20.0'
+    
+    gRpcVersion = '1.8.0'
+    slf4JVersion = '1.7.25'
+    jUnitVersion = '4.12'
 
-import org.spine3.server.BoundedContext;
-import org.spine3.server.aggregate.AggregateRepository;
+    mavenRepository = 'http://maven.teamdev.com/repository/spine'
```
Comment (armiol) [inline ext.gradle]: Can we please have repos declared after all versions?

File: hello-world/src/main/java/io/spine/helloworld/HelloWorldApp.java [hunk 2]
```diff
@@ -50,23 +54,24 @@
...
                           @Override
-                          public void onNext(Response value) {
+                          public void onNext(Ack value) {
                               System.out.println("Command posted: " + TextFormat.shortDebugString(value));
                           }
 
                           @Override
                           public void onError(Throwable t) {
-                              throw Exceptions.wrappedCause(t);
+                              throw new RuntimeException(t);
```
Comment (armiol) [inline hello-world/src/main/java/io/spine/helloworld/HelloWorldApp.java]: There is `Exceptions.newIllegalStateException(...)`. Please use that.

File: hello-world/src/main/java/io/spine/helloworld/HelloWorldApp.java [hunk 3]
```diff
@@ -50,23 +54,24 @@
  */
 public class HelloWorldApp {
 
-    private static final StorageFactory storageFactory = InMemoryStorageFactory.getInstance();
     private static final BoundedContext boundedContext = newBoundedContext();
```
Comment (armiol) [inline hello-world/src/main/java/io/spine/helloworld/HelloWorldApp.java]: Let's remove both static variables and inline them instead.

File: hello-world/src/main/java/io/spine/helloworld/HelloWorldApp.java [hunk 4]
```diff
@@ -178,7 +204,7 @@ private static CommandFactory newCommandFactory() {
      */
     private static IntroduceMyself newIntroduceWorld() {
```
Comment (armiol) [inline hello-world/src/main/java/io/spine/helloworld/HelloWorldApp.java]: This is weird. Can we please think of some renaming that would fit the natural language better? And the Javadoc section is ugly as well.

I know that's not the part of updating to the latest version, but we cannot leave this as-is.

File: server-examples/src/main/java/io/spine/examples/aggregate/server/Server.java [hunk 5]
```diff
@@ -74,6 +72,24 @@ public Server(StorageFactory storageFactory) {
...
+                              .setName(contextName.getValue())
+                              .setStorageFactorySupplier(storageSupplier)
+                              .setMultitenant(multitenant)
+                              .build();
+        final Server newServer = new Server(context);
+        return newServer;
+    }
+
+    public static Server newInstance(Supplier<StorageFactory> storageSupplier, boolean multitenant) {
+        final BoundedContextName contextName = BoundedContext.newName(Server.class.getSimpleName());
```
Comment (armiol) [inline server-examples/src/main/java/io/spine/examples/aggregate/server/Server.java]: Let's import `BoundedContext.newName()` statically.

File: server-examples/src/main/java/io/spine/examples/aggregate/server/StorageFactorySuppliers.java [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+                                       .setDatastore(datastore)
+                                       .setMultitenant(multitenant)
+                                       .build();
+        return of(storageFactory);
+    }
+
+    /**
+     * Creates a supplier for a provided {@linkplain StorageFactory}.
+     */
+    private static Supplier<StorageFactory> of(StorageFactory storageFactory) {
```
Comment (armiol) [inline server-examples/src/main/java/io/spine/examples/aggregate/server/StorageFactorySuppliers.java]: Can we remove this method and use `Suppliers.ofInstance()` directly (and import it statically)?

File: server-examples/src/main/proto/spine_examples/rejection/rejections.proto [hunk 7]
```diff
@@ -18,22 +18,22 @@
...
+
+option java_package = "io.spine.examples.rejection.rejections";
 option java_multiple_files = false;
 
-// Do not specify `java_outer_classname` option as we're not generating multiple Java classes for failures.
-// All failure messages will be generated as sub-classes of the `Failures` class by Protobuf compiler.
-// Then Spine Protobuf Gradle plugin for Java will generate `FailureThrowable` classes for these messages.
-// See `org.spine3.examples.failure.TaskAggregate` for using failure throwables in an aggregate.
+// Do not specify `java_outer_classname` option as we're not generating multiple Java classes for rejections.
+// All rejection messages will be generated as sub-classes of the `Rejections` class by Protobuf compiler.
```
Comment (armiol) [inline server-examples/src/main/proto/spine_examples/rejection/rejections.proto]: Let's use present tense. I.e. "...messages are generated" and "Then ... plugin generates ...".

File: server-examples/src/test/java/io/spine/examples/aggregate/server/ServerShould.java [hunk 8]
```diff
@@ -18,33 +18,63 @@
...
+        final Server server = Server.newInstance(supplier, false);
+
+        runServer(server);
+    }
+
+    /**
+     * The instruction on configuring the local Datastore environment is available
+     * <a href="https://github.com/SpineEventEngine/gae-java/wiki/Configuring-Local-Datastore-Environment">here.</a>
+     *
+     * <p>This test is marked as ignored because it can be executed only locally due to issues with running
```
Comment (armiol) [inline server-examples/src/test/java/io/spine/examples/aggregate/server/ServerShould.java]:
```
<p>This test is ignored as it relies on the local Datastore instance. Current CI environment does not provide that yet. See (the link to the GitHub issue here).
```

**Comment by mdrachuk** [comment]
@armiol PTAL.

File: hello-world/src/main/java/io/spine/helloworld/HelloWorldApp.java [hunk 9]
```diff
@@ -0,0 +1,256 @@
...
+         */
+        @Apply
+        private void event(UserGreeted event) {
+            // Does not modify the state of the aggregate.
+        }
+    }
+
+    /**
+     * {@linkplain BoundedContext} must register the repository, because an {@code AggregateRepository}
+     * is {@code CommandDispatcher}, that dispatches a command to {@code Aggregate}, that in it's turn
```
Comment (armiol) [inline hello-world/src/main/java/io/spine/helloworld/HelloWorldApp.java]: `its`.

**Comment by mdrachuk** [comment]
@armiol PTAL.

File: hello-world/src/main/java/io/spine/helloworld/HelloWorldApp.java [hunk 10]
```diff
@@ -0,0 +1,254 @@
...
+    }
+
+    /**
+     * {@linkplain io.spine.server.event.EventBus} must register the event subscriber
+     * to handle events.
+     */
+    private static class WorldGreetedSubscriber extends EventSubscriber {
+
+        /**
+         * Handles the event by printing "[user]: Hello, World!".
```
Comment (armiol) [inline hello-world/src/main/java/io/spine/helloworld/HelloWorldApp.java]: That's `username`, not `user`.

**Pull request merged**